src/utils/renameLabels.tsx

```diff
@@ -0,0 +1,284 @@
+// Helper function to get the faction name from the faction code
+export const getFactionName = (factionCode: string): string => {
+  const countryMapping: { [code: string]: string } = {
+    dak: "Deutsches Afrikakorps",
+    german: "Wehrmacht",
+    british: "US Forces",
+    american: "British Forces",
```

Comment:
Looks like typo here. @MichlF 

Also this whole file can have just `.ts` we don't need `.tsx`